apps/assets/automations/base/manager.py

```diff
@@ -340,6 +346,7 @@ def check_automation_enabled(self, platform, assets):
         if not platform.automation or not platform.automation.ansible_enabled:
             print(_("  - Platform {} ansible disabled").format(platform.name))
             self.on_assets_not_ansible_enabled(assets)
+            return False
 
         automation = platform.automation
 
```

Comment:
No significant code differences found between the given code snippet and its reference versions from 2021 to 2025. No issues nor suggestions can be provided based on the information available here. Any discrepancies in functionality or execution logic should be verified independently with real-world results and updated against actual system configurations.